README.md

```diff
@@ -131,13 +131,9 @@ examples of how to make this call.
 
 
 ## More information/Support
-For more information and support, refer to the competition guidelines which you should have received when registering
-for the contest, or post your questions in the support Slack channel, to which you should have also gained access 
-upon registering.
+For more information and support, refer to the competition guidelines or post your questions in the support Slack channel, both of which you should have gained access when you create a login in the competition platform.
```

Comment:
`should have gained access *to*` -- minor typo

Reply:
Oops; I've rephrased it to fix it (I think).